website/static/js/myProjects.js

```diff
@@ -1810,6 +1786,7 @@ var Information = {
         }
         if (args.selected().length === 1) {
             var item = args.selected()[0].data;
+            var permission = item.attributes.current_user_permissions[item.attributes.current_user_permissions.length-1];
```

Comment:
Cool. In that case, let's use [this idiom](http://stackoverflow.com/a/12099341) to avoid such a long line here. :-)